src/common/cockpitchannel.c

```diff
@@ -515,6 +515,16 @@ cockpit_channel_real_prepare (CockpitChannel *channel)
                                 "channel has invalid \"binary\" option: %s", binary);
         }
     }
+
+  /*
+   * The default here, can change from FALSE to TRUE over time once we assume that all
+   * cockpit-ws participants have been upgraded sufficiently. The default when we're
+   * on the channel creation side is to handle flow control.
```

Comment:
Probably once we make this decision, we can just revert this patch, in fact.  I can't imagine we'll ever have someone sending "{"flow-control": false}".